python/cudf_polars/tests/expressions/test_slice.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import polars as pl
+
+from cudf_polars.testing.asserts import assert_gpu_result_equal
+
+
+def test_slice():
+    df = pl.LazyFrame({"a": [0, 1, 2, 3], "b": [1, 2, 3, 4]})
+    q = df.select(pl.col("a").slice(1))
```

Comment:
Could you also test a negative start offset e.g `slice(-1)`?

Reply:
Done